src/devices/Pcd8544/Pcd8544.cs

```diff
@@ -0,0 +1,487 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.Gpio;
+using System.Device.Spi;
+using System.Device.Pwm;
+using System.Threading;
+using System.Drawing;
+using Iot.Device.Display.Pcd8544Enums;
+
+namespace Iot.Device.Display
+{
+    /// <summary>
+    /// PCD8544 - 48 × 84 pixels matrix LCD
+    /// </summary>
+    public class Pcd8544 : IDisposable
+    {
+        /// <summary>
+        /// Size of the screen 48 x 84 / 8
+        /// </summary>
+        public const int ScreenSizeBytes = 504;
+
+        /// <summary>
+        /// The size of the screen in terms of characters
+        /// </summary>
+        public static Size Size => new Size(84, 6);
+
+        /// <summary>
+        /// The size of the screen in terms of characters
```

Comment:
```suggestion
        /// The size of the screen in terms of pixels
```

Reply:
I'm actually confused here. The size looks similar to what you have for characters but at the same time name suggests pixels

Reply:
good catch